ni_measurementlink_service/measurement/service.py

```diff
@@ -183,10 +182,54 @@ def convert_value_to_str(value: object) -> str:
 
         self.configuration_parameter_list: List[Any] = []
         self.output_parameter_list: List[Any] = []
-        self.grpc_service = GrpcService()
-        self.context: MeasurementContext = MeasurementContext()
-        self.channel_pool: GrpcChannelPool = GrpcChannelPool()
-        self.discovery_client: DiscoveryClient = DiscoveryClient()
+        self.context = MeasurementContext()
```

Comment:
I notice you're omitting the type hints for a couple of these. Is this like C# where it's ok to use var if it's obvious from the rest of the line what the type is?

Reply:
Type checkers infer the attribute type based on the initialization code. https://mypy.readthedocs.io/en/stable/class_basics.html#instance-and-class-attributes

> ```
> class A:
>     def __init__(self, x: int) -> None:
>         self.x = x  # Aha, attribute 'x' of type 'int'
> ```